src/components/TabsPage/TabsPage.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import classNames from 'classnames';
+import { NavLink, useParams } from 'react-router-dom';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[]
+};
+
+export const TabsPage: React.FC<Props> = ({ tabs }) => {
+  const { tabId = '' } = useParams();
+  const currTab = tabs.find((tab) => tab.id === tabId);
+
+  return (
+    <>
+      <h1 className="title">Tabs page</h1>
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              data-cy="Tab"
+              key={tab.id}
+              className={classNames({ 'is-active': tabId === tab.id })}
+            >
+              <NavLink
```

Comment:
reuse it
![image](https://github.com/mate-academy/react_tabs-with-router/assets/91411692/a48438b9-a7e6-4e40-a85b-6a8bcb64ff08)